build.sbt

```diff
@@ -104,3 +104,18 @@ lazy val database = (projectMatrix in file("database"))
     ): _*
   )
   .addSingleScalaBuild(Setup.serverAndDbScalaVersion, Dependencies.databaseDependencies)
+
+/**
+ * Module `info` is the library to be plugged into application which wants to easily read the measured data stored in the
+ * server
+ */
+lazy val info = (projectMatrix in file("info"))
```

Comment:
the only thing is that I'm not sure about the name `info`; what about:

* `agent` -> `agent-publisher`, `metrics-publisher`, `measurement-publisher`, `measure-publisher`, `publisher`, or something like that, possibly instead of `publisher` we can have: `writer` or `producer` ?
* `info` -> `agent-reader`, `metrics-reader`, `measurement-reader`, `measure-reader` or `reader` or something like that, possibly instead of `reader` we can have: `consumer` or `observer`?

Reply:
even a simple `agent-publisher` and `agent-consumer` or `agent-writer` and `agent-reader` can be a nice alternative

Reply:
Let's discuss the name on the team tech call.

Reply:
We agree on _Reader_ (_Atum-Reader_)